### ISSUE TITLE
训练input_img和target_img的问题

### ISSUE THREAD

**Issue by terry2100** [hobart07/Step1X-Edit_train#2]
你好，你的训练代码中，input_img是叉子，output_img是勺子，但prompt是Replace the spoon with a fork.
是不是有问题啊

**Comment by terry2100** [comment]
noisy_model_input = (1.0 - sigmas) * ref_images + sigmas * noise

        # Prepare inputs for denoising
        x = torch.cat([noisy_model_input, noisy_model_input], dim=0).to(self.device)
        ref_images = torch.cat([ref_images, obj_images], dim=0).to(self.device)
        ref_images_raw = torch.cat([ref_images_raw, obj_images_raw], dim=0).to(self.device)
        inputs = self.prepare([prompt, negative_prompt], x, ref_image=ref_images, ref_image_raw=ref_images_raw)

这里的逻辑好像不太对吧？
